kms.tf

```diff
@@ -1,9 +1,9 @@
-# Master Account
+# Management Account
 
 module "kms_key" {
   source              = "github.com/schubergphilis/terraform-aws-mcaf-kms?ref=v0.2.0"
   name                = "inception"
-  description         = "KMS key used in the master account"
+  description         = "KMS key used in the management account"
```

Comment:
Maybe not the scope of this PR, but if 'master' is no longer used as the name, there are other references in the module that need to be changed.

Reply:
I agree, reverted this part back will refactor this in a seperate pr